src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -250,6 +250,19 @@
                         [hideLabelMobile]="false"
                         [routerLink]="['/courses', courseId, 'exercises', exercise.type + '-exercises', exercise.id, 'participate', gradedParticipation!.id]"
                     ></button>
+                    @if (exercise.allowFeedbackRequests && athenaEnabled && exercise.type === ExerciseType.TEXT) {
+                        <button
+                            class="btn btn-primary"
+                            (click)="requestFeedback()"
+                            [class.btn-sm]="smallButtons"
+                            [disabled]="gradedParticipation?.submissions?.length == 1 && !gradedParticipation!.submissions![0].submitted"
+                            [id]="'request-feedback-' + exercise.id"
+                            [ngbTooltip]="'artemisApp.exerciseActions.requestAutomaticFeedbackTooltip' | artemisTranslate"
+                        >
+                            <fa-icon [icon]="faPenSquare" [fixedWidth]="true" />
+                            <span class="d-none d-md-inline" jhiTranslate="artemisApp.exerciseActions.requestAutomaticFeedback">Send automatic feedback request</span>
```

Comment:
```suggestion
                            <span class="d-none d-md-inline" jhiTranslate="artemisApp.exerciseActions.requestAutomaticFeedback"></span>
```
You don't have to add this